lib/snitch-core/src/lib.rs

```diff
@@ -227,6 +243,7 @@ impl<const N: usize> Store<N> {
         // full. If we are able to recover here, but do _not_ have enough bytes
         // free for the next incoming record, then we'd just kick back into loss
         // state.... necessitating another loss record. And so forth.
+        index as usize
```

Comment:
This function says that it returns the number of records that were discarded, but has discarded records from `0..=index`, i.e. `index + 1` records.

Also, do we actually need this return value?  It looks like it was added in the PR, but is unused.

Reply:
Okay, I'm fine keeping it if the off-by-one is fixed!